ui/src/environments/index.ts

```diff
@@ -41,8 +41,9 @@ export interface Environment {
         readonly CONTROLLER_API_REST_READ: string,
         readonly CONTROLLER_API_REST_READWRITE: string,
 
-        readonly SETTINGS_ALERTING: string,
-        readonly SETTINGS_NETWORK_CONFIGURATION: string,
+        readonly SETTINGS_ALERTING: string | null,
+        readonly SETTINGS_NETWORK_CONFIGURATION: string | null,
+        readonly EVCS_CLUSTER: string,
```

Comment:
there is this attribute on theme.ts. but in this file this attribute is missing. so I added it.

Reply:
great 👍